documents/design/cone_of_shame.md

```diff
@@ -0,0 +1,39 @@
+# Project Title
+
+*Issue #Number*
+571
+
+**Author:**
+- Jiajun Mao
+
+## The Problem
+
+In last year's IGVC competition, Jessie turned 180 degrees before starting traveling on the course. This is presumably caused by the current navigation model ignoring everything behind Jessie and default that nothing exists there. This caused the problem with the path planning algorithm thinking that the shortest path is to walk backward to the place where "no line exists".
+
+## Proposed Solution
+
+- The problem could potentially be solved by creating a cone behind the robot and make the robot think that that is a part of the line defining the course. Thus the robot would not travel backwards.
+- Create a new layer in the 2D cost map and artificially insert points to act as a cone-shaped obstacle behind the robot
+    - Create a new layer and artificially create point behind the robot
```

Comment:
Give a bit more detail about how this is going to work, ie.

When are we going to create points behind the robot? Are we going to remove them?
What shape are these points going to be? How large?

Reply:
Details have been updated